src/main/java/com/fundation/logic/model/MetadataExtractor.java

```diff
@@ -0,0 +1,128 @@
+            while ((frameRate = stdInput.readLine()) != null) {
+                if ((frameRate.contains("Video Frame Rate"))) {
+                    int initIndex = frameRate.indexOf(":");
+                    int endIndex = frameRate.length();
+                    frameRate = frameRate.substring(initIndex + 2, endIndex);
+                    frameRate = Float.toString(Math.round(Float.parseFloat(frameRate)));
+                    frameRate = frameRate.substring(0, 2);
+                    return frameRate;
+                }
+            }
+            System.exit(0);
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.exit(-1);
+        }
+        return null;
+    }
+
+    /**
+     * This method returns metadata file type.
+     */
+    public String getFileType(File path) {
+        String videoFileType = null;
+        try {
+            Process extractMetadata = Runtime.getRuntime().exec("resources/exiftool.exe " + path.toString());
+            BufferedReader stdInput = new BufferedReader(new InputStreamReader(extractMetadata.getInputStream()));
+            while ((videoFileType = stdInput.readLine()) != null) {
+                if ((videoFileType.contains("File Type"))) {
+                    int initIndex = videoFileType.indexOf(":");
+                    int endIndex = videoFileType.length();
+                    videoFileType = videoFileType.substring(initIndex + 2, endIndex);
+                    System.out.println(videoFileType);
+                    return videoFileType;
+                }
+            }
+            System.exit(0);
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.exit(-1);
+        }
+        return null;
+    }
+
+    /**
+     * This method returns metadata resolution.
+     */
+    public String getResolution(File path) {
+        String resolution = null;
+        try {
+            Process extractMetadata = Runtime.getRuntime().exec("resources/exiftool.exe " + path.toString());
+            BufferedReader stdInput = new BufferedReader(new InputStreamReader(extractMetadata.getInputStream()));
+            while ((resolution = stdInput.readLine()) != null) {
+                if ((resolution.contains("Source Image Height"))) {
+                    System.out.println(resolution);
+                    int initIndex = resolution.indexOf(":");
+                    int endIndex = resolution.length();
+                    resolution = resolution.substring(initIndex + 2, endIndex);
+                    System.out.println(resolution);
+                    return resolution;
+                }
+            }
+            System.exit(0);
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.exit(-1);
+        }
+        return null;
+    }
+
+    /**
+     * This method returns duration of the video.
+     */
+    public String getDuration(File path) {
```

Comment:
The variable called path could have a more descriptive name, because the parameter is not a path itself, it is a file